appveyor.yml

```diff
@@ -293,9 +293,13 @@ build_script:
       )))))
 
 test_script:
+    - cmd: set TFLAGS=%DISABLED_TESTS%
     - if %TESTING%==ON (
-        echo APPVEYOR_API_URL=%APPVEYOR_API_URL% &&
-        bash.exe -e -l -c "cd /c/projects/curl/tests && ./runtests.pl -a -b$(($(echo '%APPVEYOR_API_URL%' | cut -d'/' -f3 | cut -d':' -f2)+1)) -p !flaky %DISABLED_TESTS%" )
+        if %BUILD_SYSTEM%==CMake (
+          cmake --build . --target test-nonflaky
+        ) else (
+          echo APPVEYOR_API_URL=%APPVEYOR_API_URL% &&
+          bash.exe -e -l -c "cd /c/projects/curl/tests && ./runtests.pl -a -b$(($(echo '%APPVEYOR_API_URL%' | cut -d'/' -f3 | cut -d':' -f2)+1)) -p !flaky %DISABLED_TESTS%" ))
```

Comment:
Could the `%TFLAGS%` variable be re-used here?

Reply:
TFLAGS equals DISABLED_TESTS... So no matter what to use. 
Are you suggest using a [specific trick for CMake](https://github.com/curl/curl/blob/master/tests/runtests.pl#L5183)? IMO it's not a good idea.